README.md

```diff
@@ -88,17 +88,11 @@ the alternatives you tried before submitting a PR.
 
 # Reporting security issues
 
-The maintainers take security seriously. If you discover a security 
-issue, please bring it to their attention right away!
-
-Please DO NOT file a public issue, instead send your report privately
-to security@docker.com.
-
-Security reports are greatly appreciated and we will publicly thank you 
-for it. We also like to send gifts—if you're into Docker schwag, make 
-sure to let us know. We currently do not offer a paid security bounty 
-program, but are not ruling it out in the future.
+Guidelines for reporting security issues are [here][security].
```

Comment:
Yup, see it in action [here][1].

[1]: https://github.com/wking/go-digest/blob/beb3ba70cafaf7320a4732146a9d6e82cd04690f/README.md#reporting-security-issues